internal/search/unindexed/structural.go

```diff
@@ -111,28 +111,28 @@ func runStructuralSearch(ctx context.Context, args *search.SearcherParameters, r
 	}
 
 	// For structural search with default limits we retry if we get no results.
-	fileMatches, stats, err := streaming.CollectStream(func(stream streaming.Sender) error {
-		return streamStructuralSearch(ctx, args, repos, stream)
-	})
+	agg := streaming.NewAggregatingStream()
+	err := streamStructuralSearch(ctx, args, repos, stream)
```

Comment:
```suggestion
	agg := streaming.NewAggregatingStream()
	err := streamStructuralSearch(ctx, args, repos, agg)
```

no? `agg` looks like it doesn't matter otherwise, which feels weird

Reply:
Aah! Good catch. Thank you 💯

Reply:
I'm surprised this wasn't caught by tests 🤔